app/src/main/java/org/fossasia/openevent/general/search/SearchLocationFragment.kt

```diff
@@ -95,14 +99,17 @@ class SearchLocationFragment : Fragment() {
 
     private fun redirectToMain() {
         val startMainActivity = Intent(activity, MainActivity::class.java).setFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP)
```

Comment:
It is good to use only navigation instead of intent to the activity again.

Reply:
I didn't change this. It's from previous work. I added Navigation for search. But will do that 👍

Reply:
Yes, because now we are using single activity, intents are not required.